gamedb.yaml

```diff
@@ -4019,6 +4019,19 @@
   id: 860950
   status: verified
 
+- name: "Marvel's Guardians of the Galaxy"
+  platform: epic-store
+  id: 63a665088eb1480298f1e57943b225d8
+  status: unsupported
+  notes:
+    - Launches when installed via Heroic Games Launcher, but has mediocre controller support (tested with the Steam Controller)
+
+- name: "Marvel's Guardians of the Galaxy"
+  platform: steam
+  id: 1088850
+  status: verified
+  launch_options: mv ../runtime/3C5412CE8B28C5D133F4024887AA8579.pc_binkvid ../runtime/3C5412CE8B28C5D133F4024887AA8579.pc_binkvid.disabled; ln ../runtime/0E3B3BA8C5C9E2E58AF1AFF20B899860.pc_binkvid ../runtime/3C5412CE8B28C5D133F4024887AA8579.pc_binkvid; %command%
```

Comment:
What are these launch options doing/fixing?

Reply:
This replaces the annoying intro videos with the loading animation which leads to a slightly faster startup of the game.

Reply:
I think it is better to leave the videos as is.